quay-enterprise/build-support.md

```diff
@@ -45,15 +45,15 @@ Use the environment variable `SERVER` to tell the worker the hostname at which Q
 Here's what the full command looks like:
 
 ```sh
-docker run --restart on-failure -e SERVER=wss://myquayenterprise -v /var/run/docker.sock:/var/run/docker.sock quay.io/coreos/quay-builder:v2.3.4
+docker run -it -e SERVER=wss://myquayenterprise -v /var/run/docker.sock:/var/run/docker.sock --entrypoint /bin/sh quay.io/coreos/quay-builder:v2.3.4 -c "/usr/sbin/update-ca-certificates ; quay-builder"
```

Comment:
Why add it to the non-custom CA case? If we're going to have it on both we should just add it to an entrypoint script for the container to always run it.